src/main/kotlin/com/sendy/application/usecase/auth/command/InvalidateExistingTokensUseCaseImpl.kt

```diff
@@ -0,0 +1,20 @@
+package com.sendy.application.usecase.auth.command
+
+import com.sendy.application.usecase.auth.interfaces.InvalidateExistingTokensUseCase
+import com.sendy.domain.auth.token.service.JwtTokenStorageService
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class InvalidateExistingTokensUseCaseImpl(
+    private val jwtTokenStorageService: JwtTokenStorageService,
+) : InvalidateExistingTokensUseCase {
+    
+    private val logger = LoggerFactory.getLogger(InvalidateExistingTokensUseCaseImpl::class.java)
```

Comment:
저는 개인적으로 `this`를 선호하는편입니다!

개인적인 의견이여서 댓글로 남기고 resolve 처리하겠습니다!
```suggestion
    private val logger = LoggerFactory.getLogger(this::class.java)
```